api/utils/keys/yubikey_test.go

```diff
@@ -39,7 +45,8 @@ func TestGetOrGenerateYubiKeyPrivateKey(t *testing.T) {
 	require.NoError(t, err)
 
 	// Test creating a self signed certificate with the key.
-	_, err = selfSignedTeleportClientCertificate(priv, priv.Public())
+	digest := []byte{100}
```

Comment:
This looks rather off, even though I get that it is fake. Are there not size requirements for the digest? I'd expect 32 bytes or something like that.

Eg:

```suggestion
	digest := make([]byte, 32)
```

Reply:
I meant to do `make([]byte, 100)` :facepalm: but I don't think it really matters. I'll use 32.